core/runtime/src/main/java/io/quarkus/runtime/Quarkus.java

```diff
@@ -235,20 +219,22 @@ public static void manualInitialize() {
             return;
         synchronized (manualLock) {
             tmpState = manualState;
-            if (tmpState > MANUAL_BEGIN)
-                return;
             if (tmpState == MANUAL_FAILURE)
                 throw new RuntimeException("Quarkus manual bootstrap failed");
+            if (tmpState > MANUAL_BEGIN)
+                return;
             manualState = MANUAL_BEGIN_INITIALIZATION;
```

Comment:
Was this also left accidentally?

Reply:
No, it wasn't.  manualInitialize() and manualStartup() are still used.  Or did you mean something else?